packages/flutter/lib/src/animation/curves.dart

```diff
@@ -522,16 +522,295 @@ class Curves {
   /// {@animation 464 192 https://flutter.github.io/assets-for-api-docs/assets/animation/curve_ease_in.mp4}
   static const Cubic easeIn = Cubic(0.42, 0.0, 1.0, 1.0);
 
+  /// A cubic animation curve that starts slowly and ends quickly. This is
+  /// similar to [Curves.easeIn], but with sinusoidal easing for a slightly less
+  /// abrupt beginning and end. Nonetheless, the result is quite gentle and is
+  /// hard to distinguish from [Curves.linear] at a glance.
+  ///
+  /// Derived from Robert Penner’s easing functions.
+  ///
+  /// {@animation 464 192 https://flutter.github.io/assets-for-api-docs/assets/animation/curve_ease_in_sine.mp4}
```

Comment:
If you'd like to help with it, it'd be nice to add these new videos to the class doc on like 489 too :D

Reply:
Oops! Will submit a PR shortly :+1: